src/core/main.go

```diff
@@ -146,6 +150,39 @@ func quotaSync() error {
 	return nil
 }
 
+func initClairAdapter(clairAdapterEndpoint string) error {
```

Comment:
Move to the `/pkg/scan` package, `main` is not a good place for it. We should keep the `main` as simple as possible.

Reply:
Moved